### ISSUE TITLE
Support Dynamic parameters

### ISSUE THREAD

**Issue by sergio-sastre** [cashapp/burst#65]
Any way to support Dynamic parameters similar to what TestParameterInjector **does?**
https://github.com/google/TestParameterInjector?tab=readme-ov-file#advanced-usage

**Comment by JakeWharton** [comment]
No. Not without `const fun` support by the Kotlin language first.

**Comment by swankjesse** [comment]
I think we’re blocked until kotlin.test supports dynamic tests. And at that point, that might be sufficient on its own.
